### PR TITLE
Create ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61069](https://cumulocity.atlassian.net/browse/MTM-61069)
Original PR: [7241](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7241)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: info gauge widget should support device type dashboards (#7241)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md]: The Claude AI suggested to use the following title:
```suggestion
title: Info gauge widget now supports device type dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61069
+version: 1020.29.1
+---
+info gauge widget should support device type dashboards (#7241)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md]: The Claude AI suggested to use the following body:
```suggestion
Previously, the "Info gauge" widget did not support device type dashboards, limiting its usability for users who rely on these dashboards. With this change, the "Info gauge" widget has been updated to fully support device type dashboards. This enhancement allows users to effectively utilize the "Info gauge" widget within their device type dashboard setups, providing them with a more comprehensive and flexible monitoring experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: info gauge widget should support device type dashboards (#7241)
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
```
Comment (bozhidardedov) [inline content/change-logs/application-enablement/ui-c8y-1020-29-1-info-gauge-widget-should-support-device-type-dashboards-7241.md:10]:
```suggestion
  - value: component-YbYJ3gLU_
    label: Web SDK
```

**Review by BeateRixen — APPROVED** [review]

**Review by reey — APPROVED** [review]

**Pull request merged**